requirements.txt

```diff
@@ -23,8 +23,7 @@ flake8==4.0.1
 pep8==1.7.1
 pyflakes==2.4.0
 pycodestyle==2.8.0
-python-Levenshtein==0.12.2
```

Comment:
Hey @AmoghSInamdar! Is there a reason for the switch from python-Levenshtein to Levenshtein? I see that we would also need rapidfuzz as a dependency so wanted to make sure before adding it.

Reply:
Hi @ndittren! For some reason, `python-Levenshtein` wasn't getting installed in my local environment (Python 3.9.7 on Pycharm Community). It may just be an IDE issue so you can ignore this change and keep `python-Levenshtein` instead of changing it. Hope that helps!
I haven't fixed the 'implication as disjunction' loops yet, but I've figured out the issue (part of it is just the quality of the heuristic, part of it will be a tricky fix). I'll make a separate PR for that change.

Reply:
Great, thank you for the update!